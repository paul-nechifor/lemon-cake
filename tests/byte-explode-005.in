(byte-explode :(4 0x55443322) :(1 4))
