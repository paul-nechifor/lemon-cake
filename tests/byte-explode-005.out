(34 51 68 85 4)
